addons/account/models/account_move.py

```diff
@@ -2127,6 +2141,7 @@ class AccountMoveLine(models.Model):
         index=True, ondelete="cascade",
         domain=[('deprecated', '=', False)])
     account_internal_type = fields.Selection(related='account_id.user_type_id.type', string="Internal Type", store=True, readonly=True)
+    account_root_id = fields.Many2one(related='account_id.root_id', string="Account Root", store=True, readonly=True)
```

Comment:
I don't really like the idea writing a code on journal will write on all journal items... (performance...)

Reply:
You have the field `account_internal_type` which is even worse because it is related to `account.account.type`. You already have to join/fetch/whatever on account_id, so...